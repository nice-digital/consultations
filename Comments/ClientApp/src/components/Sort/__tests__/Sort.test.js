import React from "react";
import { render  } from "@testing-library/react";
import { Sort } from "./../Sort";

test("should match the snapshot with supplied data", () => {
	const {container} = render(<Sort sortOrder="TestOrder" path="test-path" />);
	expect(container).toMatchSnapshot();
});
