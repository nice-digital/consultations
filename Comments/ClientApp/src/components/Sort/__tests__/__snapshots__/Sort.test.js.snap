// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot with supplied data 1`] = `
<div>
  <span
    aria-hidden="true"
  >
    Sort by: 
  </span>
  <a
    class="gtm-topic-list-sort-link"
    href="test-path?Sort=DocumentAsc"
  >
    <span
      class="visually-hidden"
    >
      Sort by
    </span>
     
    Document order
  </a>
  <span
    aria-label="or"
  >
     | 
  </span>
  <a
    class="gtm-topic-list-sort-link"
    href="test-path?Sort=DateDesc"
  >
    <span
      class="visually-hidden"
    >
      Sort by
    </span>
     
    Date
  </a>
</div>
`;
